client/src/components/SiteSearch/SiteSearchSidebar/SiteSearchSidebar.jsx

```diff
@@ -31,43 +32,49 @@ export const SiteSearchSidebarItem = ({
   );
 };
 
+const SiteSearchSidebarItemPill = ({ count }) => {
+  return (
+    <div className={`search-count-pill`} data-testid="count-pill">
+      <Pill>{count.toString()}</Pill>
+    </div>
+  );
+};
+
 const SiteSearchSidebar = ({ authenticated, schemes, results, searching }) => {
   const queryParams = queryStringParser.parse(useLocation().search);
   // Reset pagination on browse
   const query = queryStringParser.stringify({ ...queryParams, page: 1 });
+
+  const sidebarItems = [
+    {
+      to: `/search/cms/?${query}`,
+      label: 'Web Content',
+      iconName: 'browser',
+      disabled: false,
+      hidden: false,
+      children: <SiteSearchSidebarItemPill count={results.cms.count} />,
+    },
+    {
+      to: `/search/public/?${query}`,
+      label: 'Public Files',
+      iconName: 'folder',
+      disabled: false,
+      hidden: !schemes.includes('public'),
+      children: <SiteSearchSidebarItemPill count={results.public.count} />,
+    },
+    {
+      to: `/search/community/?${query}`,
+      label: 'Community Data',
+      iconName: 'folder',
+      disabled: false,
+      hidden: !authenticated && !schemes.includes('community'),
```

Comment:
I think this would be an or here
```suggestion
      hidden: !authenticated || !schemes.includes('community'),
```

Reply:
Yes I think your suggestion would be the correct logic. Thanks for catching that!